README.rst

```diff
@@ -50,6 +50,8 @@ Get to know
 
 - `Developer Guide </doc/motr-developer-guide.md>`_
 
+- `Provisioning Guide for Building and Testing </scripts/provisioning/README.md>`_
```

Comment:
I'd add an "environment" word here to make it more clear: "Building & Testing environment".

Reply:
Fixed, directly.